security/oauth/manager/config.go

```diff
@@ -7,17 +7,13 @@ import (
 	"golang.org/x/oauth2/clientcredentials"
 )
 
-type Endpoint struct {
-	TokenURL string `envconfig:"TOKEN_URL" env:"TOKEN_URL"`
-}
-
 type Config struct {
```

Comment:
This is used for service client configuration what is not clear from the import.
File should be moved to `security/oauth/service/client/config.go`. Otherwise developer going through the kit won't be sure what purpose does this config serve.

Reply:
@ondrejtomcik @jkralik
Do I have to move all files of security/oauth/manager to security/oauth/service/client folder?

Reply:
Yes. Correct.